README.md

```diff
@@ -1,5 +1,11 @@
 # libasynql <img src="https://poggit.pmmp.io/ci.badge/poggit/libasynql/libasynql" align="right"/>
 Asynchronous SQL access library for PocketMine plugins.
+## Why should I use this library and what asynchronous means?
+When excuting a SQL query at the main thread, **there will be a delay made** by MySQL for communicating between the database server or Sqlite for interacting with the file system. And the delay will completely block the main thread and **causes lag to the server**.
+
+Libasynql uses **different threads for excuting the queries** so the main thread will not be affect by the lag!
```

Comment:
```suggestion
Libasynql uses **different threads for executing the queries** so the main thread will not lag!
```

Reply:
Alright understandable! I will use "on" instead of "at".